JS/teamData.js

```diff
@@ -0,0 +1,121 @@
+const data = [
```

Comment:
- [ ] Please take a look at your images, as they are not being shown on the screen. Right now only the `alt` text is being shown instead of the image itself. Please work on this issue.